examples/with-vuetify/vuetify.options.js

```diff
@@ -0,0 +1,19 @@
+import colors from 'vuetify/es5/util/colors'
+
+export default {
+  customVariables: ['~/assets/variables.scss'],
```

Comment:
`customVariables` is a top-level moduleOption (https://github.com/nuxt-community/vuetify-module#customvariables) meanwhile `theme` is a framework option.

Which means that `customVariables` should still sit in `nuxt.config.js` :)

Reply:
Thanks for the correction once again!